docs/settings/settings.md

```diff
@@ -1,91 +1,97 @@
-# Kalliope settings
-
-This part of the documentation explains the main configuration of Kalliope placed in the `settings.md` file.
-
-- [Kalliope settings](#kalliope-settings)
-  - [Triggers configuration](#triggers-configuration)
-    - [default_trigger](#default-trigger)
-    - [triggers](#triggers)
-  - [Players configuration](#players-configuration)
-    - [default_player](#default-player)
-    - [players](#players)
-  - [Speech to text configuration](#speech-to-text-configuration)
-    - [default_speech_to_text](#default-speech-to-text)
-    - [speech_to_text](#speech-to-text)
-  - [Text to speech configuration](#text-to-speech-configuration)
-    - [default_text_to_speech](#default-text-to-speech)
-    - [text_to_speech](#text-to-speech)
-  - [Hooks](#hooks)
-  - [Rest API](#rest-api)
-  - [Resources directory](#resources-directory)
-  - [Global Variables](#global-variables)
-  - [Options](#options)
-    - [energy_threshold](#energy-threshold)
-    - [adjust_for_ambient_noise_second](#adjust-for-ambient-noise-second)
-  - [Next: configure the brain of Kalliope](#next--configure-the-brain-of-kalliope)
+# settings.yml
+
+This part of the documentation explains the main configuration of Kalliope placed in the `settings.yml` file.
+
+Kalliope will look for the settings file in the order bellow:
+
+- From you current folder, E.g `/home/pi/my_kalliope/settings.yml`
+- From `/etc/kalliope/settings.yml`
+- From the default `settings.yml` located in the root of the project tree.
 
 ## Triggers configuration
 
 ### default_trigger
 
-The trigger is the module detecting the hotword that will wake up Kalliope.
-Common usage of hotword include Alexa on Amazon Echo, OK Google on some Android devices and Hey Siri on iPhones.
+The trigger is the engine in charge of detecting the hotword that will wake up Kalliope.
+Common usage of hotword include "Alexa" on Amazon Echo, "OK Google" on some Android devices and "Hey Siri" on iPhones.
 
 Specify the name of the trigger module you want to use.
-```yml
+```yaml
 default_trigger: "trigger_name"
 ```
 
-Available triggers for Kalliope are:
-- snowboy
+E.g
+```yaml
+default_trigger: "snowboy"
+```
 
 ### triggers
```

Comment:
maybe introduce in this section that "Kalliope" is the default hotword but it can be something else

Reply:
actually it depends of the used starter kit.